server/src/main/java/org/jboss/as/server/ServerEnvironmentService.java

```diff
@@ -54,9 +55,14 @@
  */
 public class ServerEnvironmentService implements Service<ServerEnvironment> {
```

Comment:
Shouldn't we use `org.jboss.msc.Service` as `org.jboss.msc.service.Service` is deprecated already ? or we need to address that in a different issue and pr ? @pferraro @yersan

Reply:
Rewriting the ServerEnvironmentService class is not the goal of WFCORE-6321.

That said, I did address this deprecated MSC usage as part of https://github.com/wildfly/wildfly-core/pull/5516.
See: https://github.com/wildfly/wildfly-core/pull/5516/commits/8ff268a719349678b45b18f5973f1f68a9858ad6
Hopefully, someone will review this (and its dependencies) soon.

Reply:
OK, I see. thanks!